src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover } from '@patternfly/react-core';
+
+import { getHostStatusText } from '../../selectors/host/selectors';
```

Comment:
can be shortened to
`import { getHostStatusText } from '../../selectors';`

Reply:
Done